lib/ansible/modules/cloud/azure/azure_rm_dnsrecordset.py

```diff
@@ -167,6 +166,76 @@
 '''
 
 RETURN = '''
+state:
+    description:
+        - Current state of the DNS record set.
+    returned: always
+    type: complex
+    contains:
+        id:
+            description:
+                - The DNS record set ID.
+            returned: always
+            type: str
+            sample: "/subscriptions/xxxx......xxx/resourceGroups/v-xisuRG/providers/Microsoft.Network/dnszones/b57dc95985712e4523282.com/A/www"
+        name:
+            description:
+                - Relate name of the record set.
+            returned: always
+            type: str
+            sample: 'www'
+        fqdn:
+            description:
+                - Fully qualified domain anme of the record set.
+            return: always
+            type: str
+            sample: www.b57dc95985712e4523282.com
+        etag:
+            description:
+                - The etag of the record set.
+            return: always
+            type: str
+            sample: 692c3e92-a618-46fc-aecd-8f888807cd6c
+        provisioning_state:
+            description:
+                - The DNS record set state.
+            return: always
+            type: str
+            sample: Succeeded
+        target_resource:
+            description:
+                - The target resource of the error message.
```

Comment:
Should this be "The target resource of the record set" instead of "of the error message"? If not, why does the target resource relate to an error message?  I don't see an `error_message` field among the return fields.